nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -1540,6 +1540,12 @@ public boolean isZooKeeperTlsConfigurationPresent() {
             && getProperty(NiFiProperties.ZOOKEEPER_SECURITY_TRUSTSTORE_PASSWD) != null;
     }
 
+    public boolean isTlsConfigurationPresent() {
+        return StringUtils.isNotBlank(getProperty(NiFiProperties.SECURITY_KEYSTORE))
+            && getProperty(NiFiProperties.SECURITY_KEYSTORE_PASSWD) != null
+            && StringUtils.isNotBlank(getProperty(NiFiProperties.SECURITY_TRUSTSTORE));
```

Comment:
We made a stronger assumption in `isZooKeeperTlsConfigurationPresent()` that truststorePasswd was present. I think what you're doing here, allowing empty truststorePasswd, is correct but should we change `isZooKeeperTlsConfigurationPresent()`.